charts/jaeger/templates/agent-sa.yaml

```diff
@@ -6,4 +6,8 @@ metadata:
   labels:
     {{- include "jaeger.labels" . | nindent 4 }}
     app.kubernetes.io/component: agent
+{{- with .Values.agent.imagePullSecrets }}
+imagePullSecrets:
+  {{- toYaml . | nindent 0 }}
```

Comment:
```suggestion
  {{- toYaml . | nindent 2 }}
```

Reply:
Sure. I was sticking to what was rendered out in the other templates. I'll update that to `nindent 2`

Same applies for the other changes as suggested by you, as well.

Reply:
🙏 

Thanks! I'll try to clean up the lists to be indented in a future PR to avoid this confusion.